src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 7) {
+    return 230;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days === 3) {
+    return 100;
+  }
+
+  return 80;
```

Comment:
What is the magic numbers?
You should calculate cost and not return constant value.
Also check [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) - there are some hints